### PR TITLE
Added possibility to ifdef-away descriptions of namespace entries

### PR DESCRIPTION
**Pull request by eliasnaslund**
Removing descriptions of namespace entries saves a few kb.

See  Issue #1698.

### CODE REVIEW

**Comment by Pro** [comment]
@eliasnaslund can you please rebase onto master? I just fixed the appveyor issue

**Comment by eliasnaslund** [comment]
I was home sick without access to my computer last week but now I have rebased it onto master.

File: CMakeLists.txt [hunk 1]
```diff
@@ -95,6 +95,9 @@ mark_as_advanced(UA_ENABLE_STATUSCODE_DESCRIPTIONS)
 option(UA_ENABLE_TYPENAMES "Add the type and member names to the UA_DataType structure" ON)
 mark_as_advanced(UA_ENABLE_TYPENAMES)
 
+option(UA_ENABLE_NAMESPACE_DESCRIPTIONS "Add the description of namespace entry" ON)
```
Comment (Pro) [inline CMakeLists.txt]: One small change request here:
It's probably better to name it `UA_ENABLE_NODESET_COMPILER_DESCRIPTIONS`

Can you clarify the option a bit more:

"Set node description attribute for nodeset compiler generated nodes"

Comment (eliasnaslund) [reply CMakeLists.txt]: No problem, this is now fixed. Should I merge the two commits to one? not sure how that impacts the pull request(?).

**Comment by Pro** [comment]
We can squash merge directly on github. Otherwise it would also be possible that you interatively rebase your branch and squash the commits. But for now it's fine.

**Pull request merged**